plugins/csharp/Managed/ResultOutput.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Drawing;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Neurala {
+    public class ResultOutput {
+        [StructLayout(LayoutKind.Sequential)]
+        private struct StructureForInvoke {
+            public IntPtr Metadata;
+            public IntPtr ImageData;
+            public UInt64 Width;
+            public UInt64 Height;
+        }
+
+        public static void Invoke(IntPtr argument, int size) {
+            var structure = Marshal.PtrToStructure<StructureForInvoke>(argument);
+            var metadata = Marshal.PtrToStringUTF8(structure.Metadata);
+
+            /*
+            var stream = new MemoryStream(structure.ImageData);
+            var image = new Bitmap(stream);
+
+            Console.WriteLine($"\n{metadata}\n\n{image.Width}x{image.Height} {image.PixelFormat}");
+            */

```

Comment:
Wasn't sure how to load an image from just a pointer, without copying, so I may have to marshal it into a `byte[]`, instead.